src/Services/DocumentService.php

```diff
@@ -111,7 +111,8 @@ public function getAllAtPath($pid, $path)
         $fileResults = array();
         while ($row = sqlFetchArray($documentResults)) {
             array_push($fileResults, array(
-                "filename" => basename($row["url"]),
```

Comment:
yep, above is from the old days when filenames were the document names (until they then became encrypted files with uuid names).